### ISSUE TITLE
GitHub Organization Avatar

### ISSUE THREAD

**Issue by msgilligan** [hackthefuture/hackthefuture.github.com#13]
I created an Avatar for the [Hack the Future GitHub organization](https://github.com/hackthefuture) using my Google Apps account using a `+htf` suffix on my Google Apps email address.

I used the following image: https://github.com/hackthefuture/hackthefuture.github.com/blob/master/images/hackthefuture2-2.jpg

Someone should probably change this to use an official HtF email account.  I don't mind HtF continuing to use mine, but I thought I'd mention it here.  (In case anyone is worried that I might turn evil.)

**Comment by jdonald** [comment]
I looked into this and at first I changed the "Gravatar email (private)" field to `partners@hackthefuture.org`

Then I looked into making a Gravatar account for us, but now I don't think that's necessary. Gravatar is one way to automatically locate the avatar but the profile picture can be uploaded independently as well. I've thus changed the Gravatar email to blank in our profile, and that seems to be fine.
